### ISSUE TITLE
Installation Issue in Python 3.11

### ISSUE THREAD

**Issue by mmmarinho** [quadprog/quadprog#41]
Hello!

It might be a peculiarity of my system (for which I apologize in advance), but I was getting the following error when trying to install from PyPI in Python 3.11.

```bash
clang -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -I/Users/murilomarinho/git/dqrobotics/quickandsafe/python/venv/include -I/opt/homebrew/opt/python@3.11/Frameworks/Python.framework/Versions/3.11/include/python3.11 -c quadprog/quadprog.c -o build/temp.macosx-13-arm64-cpython-311/quadprog/quadprog.o
      quadprog/quadprog.c:198:12: fatal error: 'longintrepr.h' file not found
        #include "longintrepr.h"
                 ^~~~~~~~~~~~~~~
```

This error seems related to changes in [Cython for Python 3.11](https://docs.python.org/ja/3.11/whatsnew/3.11.html).

I could fix it locally by refreshing the `quadprog.c` by running
`python -m cython quadprog/quadprog.pyx`
as you do in the CI step. 

I hope this information can be useful.

Cheers,
Murilo

**Comment by 0-wiz-0** [comment]
Related to this: the pypi source tarball does not provide the pyx file, so it can't be used for building with python 3.11.
Please add the pyx file to the pypi source tarball. Thanks!

**Comment by Biszu** [comment]
@preeyan Looks like it has been already fixed in Cython, but a new release/rebuild of quadprog is required. Would it be possible to take a look at it and hopefully add support for Python 3.11?

**Comment by TimmHess** [comment]
Friendly bump :)

**Comment by HKaras** [comment]
I have just created a pull request which replaces `quadprog.c` with `quadprog.pyx`. #43  This should allow for Python 3.11 support even without wheels being built. Though wheels should of course be built.

**Comment by stephane-caron** [comment]
For context, this issue is also faced by @sandeep026 in https://github.com/qpsolvers/qpsolvers/discussions/246. 

Any blocker with https://github.com/quadprog/quadprog/pull/43?

**Comment by HKaras** [comment]
There appears to be no interested maintainers for this package. I tried sending a traditional email to @rmcgibbo but never recieved a response. We have removed quadprog from our software package for this reason.

**Comment by sandeep026** [comment]
Thanks for the update. BTW great interface and much needed in python, developers.

**Comment by rmcgibbo** [comment]
See https://github.com/quadprog/quadprog/issues/13

I'd be happy to add you as a maintainer.

**Comment by jmoralez** [comment]
We just released [0.1.12](https://pypi.org/project/quadprog/0.1.12/#files) which has wheels for CPython 3.8-3.12. Feel free to reopen if you encounter any issue with them.
